plugins/catalog-backend-module-aws/api-report.md

```diff
@@ -27,14 +29,18 @@ export class AwsEKSClusterProcessor implements CatalogProcessor {
   constructor(options: {
     credentialsFactory?: AWSCredentialFactory;
     credentialsManager?: AwsCredentialsManager;
+    entityFromEKSCluster?: EKSClusterEntityGenerator;
   });
   // (undocumented)
-  static fromConfig(configRoot: Config): AwsEKSClusterProcessor;
+  static fromConfig(
+    configRoot: Config,
+    options: {
```

Comment:
suggestion: making `options` optional as well, you could keep the signature for the processor creation compatible.